<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveMQTestClass</class>
 <widget class="QMainWindow" name="ActiveMQTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActiveMQTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbSend">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdtSend">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>发送内容：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>接收到的内容：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRecv">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>741</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>680</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRead">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>680</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>读取</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="activemqtest.qrc"/>
 </resources>
 <connections/>
</ui>
